lib/cmd/fh3/app/envvars/list.js

```diff
@@ -0,0 +1,28 @@
+var fhc = require("../../../../fhc");
+
+module.exports = {
+  'desc' : 'Gets the deployed environment variables for a cloud app in an environment.',
+  'examples' : [
+    { cmd : 'fhc app envvars list --app=2b --env=dev', desc : 'Lists all env vars for the 2b app in the dev environment'},
+    { cmd : 'fhc app envvars list --app=2b --env=dev --deployed', desc : 'Lists the user-created and deployed env vars for the 2b app in the dev environment'}
+  ],
+  'demand' : ['app', 'env'],
+  'alias' : {
+    'app' : 'a',
+    'env' : 'e',
+    0 : 'app',
+    1 : 'env'
+  },
+  'describe' : {
+    'app' : 'Unique 24 character GUID of the app you want to retrieve env vars from',
+    'env' : 'Environment to look up the env vars for',
+    'deployed' : 'Only list user created environment variables? These have typically already been deployed to an environment'
+  },
+  'url' : function(argv){
+    if (argv.deployed){
```

Comment:
Is this right? so if we want the deployed env vars we call millicore but if we want them all we call supercore. Seems unfortunate if this is the case